### PR TITLE
Add HTML Structure

### PR DESCRIPTION
**Pull request by RickKollenbroich93**

### CODE REVIEW

**Review by Seborider — APPROVED** [review]

File: src/components/fieldset/fieldset.html [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
-</section>
+<form>
+  <section>
+    <p>
+      <label>
+        <span>Enter Task</span>
+        <abbr title="required">*</abbr>
+      </label>
+      <input type="text" id="task" name="task" />
+    </p>
```
Comment (marwinburesch) [inline src/components/fieldset/fieldset.html]: I wouldn't recommend using a `<p>` here the only thing you need is the `label` and its `input`. However, `input` should be a child element of `label`.

File: src/components/fieldset/fieldset.html [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+
+    <fieldset>
+      <legend>WHEN</legend>
+      <ul>
+        <li>
```
Comment (marwinburesch) [inline src/components/fieldset/fieldset.html]: No `ul` and `li` needed, the form and its structure already form a semantic collection of elements

File: src/components/fieldset/fieldset.html [hunk 3]
```diff
@@ -1,6 +1,38 @@
-<section>
-  <fieldset></fieldset>
-</section>
+<form>
+  <section>
```
Comment (marwinburesch) [inline src/components/fieldset/fieldset.html:27]: I don't think you need the `section` here, `form` is already a sectioning element

File: src/components/fieldset/fieldset.html [hunk 4]
```diff
@@ -1,6 +1,38 @@
...
+            Select Date
+          </label>
+        </li>
+      </ul>
+    </fieldset>
+  </section>
+</form>
 <section>
   <p>
     <button class="btn" type="submit">Add Task</button>
```
Comment (marwinburesch) [inline src/components/fieldset/fieldset.html]: That `type="submit"` won't do much if the element is outside the `form`

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Thank you for the PR :) 
Good start!

File: src/components/fieldset/fieldset.html (hunk 4 above)
Comment (RickKollenbroich93) [reply src/components/fieldset/fieldset.html]: THX :D

**Pull request merged**